tests/js/client/server_parameters/test-extended-names-collections-on.js

```diff
@@ -0,0 +1,1081 @@
+      let c = db._createEdgeCollection(extendedName);
+
+      let docs = [];
+      for (let i = 0; i < 100; ++i) {
+        docs.push({ _key: "test" + i, _from: "v/test" + i, _to: "v/test" + (i % 10), value: i });
+      }
+      c.insert(docs);
+
+      let edges = c.outEdges("v/test0");
+      assertEqual(1, edges.length);
+      assertEqual(extendedName + "/test0", edges[0]._id);
+      assertEqual(0, edges[0].value);
+      
+      edges = c.inEdges("v/test0");
+      edges.sort((l, r) => { return l.value - r.value; });
+      assertEqual(10, edges.length);
+      for (let i = 0; i < 10; ++i) { 
+        assertEqual(extendedName + "/test" + (i * 10), edges[i]._id);
+        assertEqual(i * 10, edges[i].value);
+      }
+      
+      edges = c.edges("v/test0");
+      edges.sort((l, r) => { return l.value - r.value; });
+      assertEqual(10, edges.length);
+      for (let i = 0; i < 10; ++i) { 
+        assertEqual(extendedName + "/test" + (i * 10), edges[i]._id);
+        assertEqual(i * 10, edges[i].value);
+      }
+    },
+    
+    testEdgeIndex: function() {
+      let c = db._createEdgeCollection(extendedName);
+      
+      let indexes = c.indexes();
+      assertEqual(2, indexes.length);
+
+      assertEqual("primary", indexes[0].type);
+      assertEqual("edge", indexes[1].type);
+      assertEqual(extendedName + "/2", indexes[1].id);
+    },
+  
+    testEdgeQueriesUsingExtendedVertices: function() {
+      // vertices
+      let c = db._create(extendedName);
+      let docs = [];
+      for (let i = 0; i < 100; ++i) {
+        docs.push({ _key: "test" + i });
+      }
+      c.insert(docs);
+
+      // edges
+      c = db._createEdgeCollection(traditionalName);
+      docs = [];
+      for (let i = 0; i < 100; ++i) {
+        docs.push({ _key: "test" + i, _from: extendedName + "/test" + i, _to: extendedName + "/test" + (i % 10), value: i });
+      }
+      c.insert(docs);
+
+      let res = db._query("FOR doc IN `" + traditionalName + "` FILTER doc._from == '" + extendedName + "/test0' RETURN doc").toArray();
+      assertEqual(1, res.length);
+      assertEqual(extendedName + "/test0", res[0]._from);
+      assertEqual(extendedName + "/test0", res[0]._to);
+      
+      // query again until we have a cache hit in the in-memory cache for edges
+      let tries = 0;
+      while (++tries < 60) {
+        let res = db._query("FOR doc IN `" + traditionalName + "` FILTER doc._from == '" + extendedName + "/test0' RETURN doc");
+        let stats = res.getExtra().stats;
+        res = res.toArray();
+        assertEqual(1, res.length);
+        assertEqual(extendedName + "/test0", res[0]._from);
+        assertEqual(extendedName + "/test0", res[0]._to);
+        if (stats.cacheHits > 0) {
+          break;
+        }
+        internal.sleep(0.5);
+      }
+      assertTrue(tries < 60);
+
+      res = db._query("FOR doc IN `" + traditionalName + "` FILTER doc._to == '" + extendedName + "/test0' RETURN doc").toArray();
```

Comment:
could this be passed by bind value on the second attempt?

Reply:
Done